src/app/collective-rewards/metrics/components/ABIMetrics/ABIMetrics.tsx

```diff
@@ -44,3 +42,24 @@ export const ABIMetrics = () => {
     </>
   )
 }
+
+const ABIMetricsFromChain = () => {
+  const { data: abiPct, isLoading, error } = useGetMetricsAbi()
+  useHandleErrors({ error, title: 'Error loading ABI metrics' })
+
+  return <ABIMetricsContent abiPct={abiPct} isLoading={isLoading} />
+}
+
+const ABIMetricsWTheGraph = () => {
```

Comment:
Since the graph is going to become the standard, I would rename to ABiMetrics.

Reply:
For now I'd prefer to keep it like that to have it clear what it does.